CMakeLists.txt

```diff
@@ -435,6 +435,14 @@ set(drishti_project_config "${drishti_generated_dir}/${CMAKE_PROJECT_NAME}Config
 set(drishti_targets_export_name "${CMAKE_PROJECT_NAME}Targets")
 set(drishti_namespace "${CMAKE_PROJECT_NAME}::")
 
+#### std::to_string ####
+try_compile(DRISHTI_HAVE_TO_STRING "${CMAKE_BINARY_DIR}/compile_tests" "${PROJECT_SOURCE_DIR}/cmake/to_string.cpp")
+if(DRISHTI_HAVE_TO_STRING)
+  set(DRISHTI_ADD_TO_STRING OFF)
```

Comment:
Why extra variable needed? It can be:
```cmake
if(DRISHTI_HAVE_TO_STRING)
  target_compile_definitions(... PUBLIC DRISHTI_HAVE_TO_STRING)
endif()
```

```cpp
#if !defined(DRISHTI_HAVE_TO_STRING)
#endif
```

Reply:
There's no real difference.  Using a positive condition seemed slightly more readable.  I guess DRISHTI_HAVE_TO_STRING makes the CMake a litlte shorter.  We can change it.